src/Microsoft.Health.Dicom.Core/Features/Common/LoggingWorkitemStore.cs

```diff
@@ -21,6 +21,12 @@ public class LoggingWorkitemStore : IWorkitemStore
                    default,
                    "Adding workitem '{WorkitemInstanceIdentifier}'.");
 
+        private static readonly Action<ILogger, string, Exception> LogQueryWorkitemDelegate =
```

Comment:
Feel free to not use this Decorator pattern for logging. It was Jack's preference, I did not see a need for another class just for logging, I actually like the readability of inline logs better.

Reply:
Also same here, should this more to workitem folder

Reply:
Sure, we also dont like it.